src/dsd-guide.adoc

```diff
@@ -219,9 +219,8 @@ second element (the Value) must be:
 
 * an Integer,
 * a String,
-* a Reference, or
-* a Package consisting entirely of Integer, String, or Reference objects
-  (and specifically not containing a nested Package).
+* a Reference (but not referencing a Package which in turn contains nested Package references), or
+* a Package (but not containing a reference to another Package).
```

Comment:
"A reference to a Package" is not particularly well defined as far as I can say.  If the name (or generally a namepath) of an object appears in the source code, which is regarded as a "reference" to that object, there are two cases.  If it is possible to evaluate it, it will be evaluated and the return value (whatever it is) will be used where the name has appeared.  If it cannot be evaluated, it will be held as a "reference object" (this happens if the target object is a device, for example).  Objects that evaluate to Packages are in the first category, so they are evaluated in that case and the Package appears where the name has been used.